core/tools.js

```diff
@@ -1844,6 +1844,39 @@
 				}
 				return acc;
 			}
+		},
+
+		/**
+		 * A set of object helpers.
+		 *
+		 * @property {CKEDITOR.tools.object}
+		 * @member CKEDITOR.tools
+		 */
+		object: {
+			/**
+			 * Return a first key from `obj` which has given `value`.
+			 *
+			 * @param {Object} obj Object which `key` is looked for.
+			 * @param {Mixed} value Object's `value` to be looked for.
+			 * @returns {String/null} Matched `key` or `null` if not found.
+			 * @member CKEDITOR.tools.object
+			 */
+
+			findKey: function( obj, value ) {
```

Comment:
It'd be nice to add some basic unit tests. You can look how `tests/core/tools/array.js` is done and do something similiar in `tests/core/tools/object.js`.